### PR TITLE
InfluxQL WHERE clause, date-time related content

### PR DESCRIPTION
**Pull request by sanderson**
- Documents the InfluxQL `WHERE` clause
- Documents InfluxQL time and time zone functionality
- Adds InfluxQL date and time functions
- Updates features in progress with associated issues

---

- [x] Rebased/mergeable

### CODE REVIEW

**Review by stuartcarnie — APPROVED** [review]
Very nice. Really appreciate the issue tracking links too 💯!

File: content/influxdb/cloud-dedicated/reference/influxql/feature-support.md [hunk 1]
```diff
@@ -16,12 +16,68 @@ This process is ongoing and some InfluxQL features are still being implemented.
...
+which specifies the number of [series](/influxdb/cloud-dedicated/reference/glossary/#series)
+to skip before returning results.
+
+**Tracking issue**: [influxdb_iox#6940](https://github.com/influxdata/influxdb_iox/issues/6940)
+
+### Project additional fields with selector functions
+
+InfluxQL in {{< cloud-name >}} does not currently let you project additional
+fields when using [selector functions](/influxdb/cloud-dedicated/reference/influxql/functions/selectors/).
+Selector functions return specifics row with the value that meets the criteria
```
Comment (jstirnaman) [inline content/influxdb/cloud-dedicated/reference/influxql/feature-support.md:67]:
```suggestion
Selector functions return a specific row with the value that meets the criteria
```

File: content/influxdb/cloud-dedicated/reference/influxql/time-and-timezone.md [hunk 2]
```diff
@@ -0,0 +1,374 @@
...
+{{% /note %}}
+
+## Query time range
+
+To specify the time range of a query, use conditional expressions in the 
+[`WHERE` clause](/inflxudb/cloud-dedicated/reference/influxql/where/) that
+compare the value of the `time` column to an absolute timestamp or a relative
+timestamp.
+
+- **Absolute time range**: Define query time bounds with timestamp literal
```
Comment (jstirnaman) [inline content/influxdb/cloud-dedicated/reference/influxql/time-and-timezone.md:89]:
```suggestion
- **Absolute time range**: Define query time bounds with timestamp literals
```

**Review by jstirnaman — APPROVED** [review]
👍

**Pull request merged**